pkgs/applications/networking/cluster/terraform-providers/default.nix

```
@@ -19,10 +19,11 @@ let
      , vendorSha256 ? throw "vendorSha256 missing: please use `buildGoModule`" /* added 2022/01 */
      , deleteVendor ? false
      , proxyVendor ? false
+     , localBuildGoModule ? buildGoModule
```

`local` may be a bit misleading to use for this when `nixpkgs` has things like `runCommandLocal`, `preferLocalBuild`, etc.


```suggestion
     , mkProviderGoModule ? buildGoModule
```